app/views/admin/infobase/_registered_users.html.haml

```diff
@@ -0,0 +1,14 @@
+%table.ui.very.basic.table{style: 'border: none'}
+  %thead
+    %tr
+      %th Name
+      %th E-mail
+      %th Date of Registration
+      %th.right.aligned Status
+  %tbody
+    - @users.each do |user|
+      %tr
+        %td{"data-label" => "Name"}= user.name
+        %td{"data-label" => "E-mail"}= user.email
+        %td{"data-label" => "Date of Registration"}= user.created_at.to_date
+        %td.right.aligned{"data-label" => "Status"}= user.admin ? 'Admin' : 'User'
```

Comment:
`'Admin' : 'User'` that should be constant AT LEAST.
So? Decorator, yes. You will call something like `user.role` and this particular logic will be put to `def role` in the draper